### DISCUSSION TITLE
[灌水]避雷那个什么家政公司

### DISCUSSION THREAD

**Discussion by 23156484537** [share121/inter-knot#1363] [其他]
![屏幕截图(3)](https://github.com/user-attachments/assets/f494bcef-228a-4cf8-ae93-cb52591bbe50)
今天路过六分街汀曼门口的时候看到有个穿女仆装的直接躺地上睡觉，这也太不得体了，打听了一下才知道是维多利亚家政，这家收费还高的离谱，手底下的女仆就是这样的?女仆的入职培训怎么培训的?

二编
听别人说这个长着鲨鱼尾巴的女仆和六分街影像店的灰头发店长走的很近，两人天天去光映广场逛街，就差亲嘴了，看来我上次在光映广场那边看到那个店长和几个围在那女仆旁边的几个男生吵架多半可以实锤了，女仆的私生活这么差，真不知道什么样的人才会请他们。

**Comment by Inter-Knot-User** [comment]
对！我也看到过！尤其是那个小女孩，经常拿着个电锯在那吓唬人，真的好可怕！不知道是哪个家长教育出来的！

**Comment by zzziloveyou** [comment]
还是不要小瞧了，这群人敢这么光明正大，肯定不是一般的家政，要么家底雄厚，要么打的你满地找牙，那武器铁匠看都羡慕，实力还能小吗？

**Comment by sishui-buyin** [comment]
eee我点过一次，其实还不错啦，就是那个不提供摸尾巴服务

**Comment by dengyunzhe** [comment]
违规内容

**Comment by 23156484537** [comment]
![760832b2fc3afe496f77bba6045add55](https://github.com/user-attachments/assets/7aed4a30-a2a3-48c7-bc74-eed88e266be1)
图片好像太大加载不出来，我再发一张

**Reply by EhuanyingK** [reply]
> ![760832b2fc3afe496f77bba6045add55](https://private-user-images.githubusercontent.com/95210690/356857018-7aed4a30-a2a3-48c7-bc74-eed88e266be1.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjMzNjA2NDMsIm5iZiI6MTcyMzM2MDM0MywicGF0aCI6Ii85NTIxMDY5MC8zNTY4NTcwMTgtN2FlZDRhMzAtYTJhMy00OGM3LWJjNzQtZWVkODhlMjY2YmUxLmpwZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MTElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODExVDA3MTIyM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTEyZWYyNTRiZjE4YmQyMmRjMDBkM2FkNzk2ODMxMTY5Mjc4ZDkzMTNjNDg4MjdiYjk3NWJmODMzNDE4Zjk4N2YmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.J95ro9kGwVli_okDAYsE_3TzooyWstYAWqbENJRewlE) 图片好像太大加载不出来，我再发一张

发出来了，不用再发一次

**Comment by EhuanyingK** [comment]
这绳网怎么还带诈骗的，举办了

**Comment by VacuolePaoo** [comment]
关于绳匠 **[@EhuanyingK](https://github.com/EhuanyingK)** 举报的帖子「[灌水]避雷那个什么家政公司」  id: #1363 中，包含[虚假、不准确或有意欺骗性信息](https://docs.github.com/zh/site-policy/acceptable-use-policies/github-misinformation-and-disinformation)、[淫秽](https://docs.github.com/zh/site-policy/acceptable-use-policies/github-sexually-obscene-content)或与性剥削或性虐待有关的内容。
举报信息属实，已经处理

> 请为本次服务评分：⭐⭐⭐⭐⭐

**Comment by kaijinti** [comment]
你不会是对家公司来发黑贴的吧？

**Comment by NekoWise** [comment]
这个委托我们接了

**Comment by yutong903** [comment]
新闻已经报道了，店主天天晚上出去找那个女仆约会，两人都亲上了<img src="https://bbs-static.miyoushe.com/static/2024/03/21/d0b58fba409fa41418ebb782c0ba5be8_3368113894212916236.png" alt="丽都漫步-妮可" width="75" >

**Comment by lip33625** [comment]
楼主还好吗 9楼艾莲本人回你了，现在不会已经被——

**Reply by NekoWise** [reply]
放心，楼主好好的，跟活人没两样

**Reply by lip33625** [reply]
艾莲小姐你...冒昧问下，那尾巴是...

**Reply by NekoWise** [reply]
鲨希人的尾巴，怎么了，生来就有的，就跟熊希人有熊耳一个道理

**Reply by lip33625** [reply]
那我如果指定你的话 能不能...就...稍微...
